res/values-tl/strings.xml

```diff
@@ -217,7 +217,7 @@
     <string name="GOAL_">"GOAL!:"</string>
     <string name="SOCCER">"SOCCER:"</string>
     <string name="DRAW">"PATAS:"</string>
-    <string name="Clan_Name">"Clan Name:"</string>
+    <string name="Clan_Name">"Pangalan ng Clan:"</string>
```

Comment:
Shouldn't "clan" look like this in filipino/tagalog, according to my basic knowledge?

```suggestion
    <string name="Clan_Name">"Pangalan ng Klan:"</string>

Reply:
That is the correct tagalog of the clan name and I am a Filipino

Reply:
Got it, thanks. Don't forget to mark this conversation as resolved when you can.